downloads.html

```diff
@@ -65,22 +65,27 @@ <h1>Downloading The Intel® Implicit SPMD Program Compiler</h1>
           source and binary form.
         </p>
         <p>
-          Here are latest released ISPC binaries based on LLVM 18.1.8:
+          Here are the available ISPC packages:
+        </p>
+        <ul>
+          <li>Windows (64 bit) ISPC binary</li>
+          <li>macOS (x86, 64 bit) ISPC binary</li>
+          <li>macOS (ARM64) ISPC binary</li>
+          <li>macOS (Universal Binary) ISPC binary</li>
+          <li>Linux (64 bit) ISPC binary</li>
+          <li>Linux (64 bit) ISPC binary for OneAPI</li>
+          <li>Linux (64 bit) ISPC binary for AARCH64</li>

```

Comment:
This is confusing, architecture should go with bitness, it's kinda the same / related thing (i.e. x86-64, aarch64). But `OneAPI` is a separate thing. We should probably clarify what does this mean.